### DISCUSSION TITLE
I would like to say thank you.

### DISCUSSION THREAD

**Discussion by defmans7** [SamDel/ChromeCast-Desktop-Audio-Streamer#142] [Q&A] [answered]
I really appreciate everyone that has worked on this repo. A solid solution for a niche problem. 

If there is a way to show my appreciation in a material way, please let me know - I know this isn't why we do open source work, but I would still like to donate to a cause you support (or something like that) at least. 

Big love.

**Comment by SamDel** [comment, marked as answer]
Thanks @defmans7, appreciated!

As far as I'm concerned you can give your donation to local charity of your choice (and drop a line here if you like). But maybe someone has a better idea. What do you think @FA-Bubba?

**Comment by FA-Bubba** [comment]
Sounds good to me!
